src/serac/physics/state/finite_element_vector.hpp

```diff
@@ -93,6 +93,18 @@ class FiniteElementVector {
    */
   FiniteElementVector(mfem::ParMesh& mesh, mfem::ParFiniteElementSpace& space, const std::string& name = "");
 
+  /**
+   * @brief Delete the default copy constructor
+   */
+  FiniteElementVector(const FiniteElementVector&) = delete;
+
+  /**
+   * @brief Move construct a new Finite Element Vector object
+   *
+   * @param[in] input_vector The input vector used for construction
+   */
+  FiniteElementVector(FiniteElementVector&& input_vector);
```

Comment:
I think this can just be `= default` so you don't have to implement it manually

Reply:
Since I'm getting notifications for this, I don't think this work as expected, since `mfem::HypreParVector` doesn't have a proper defined move-constructor.

I think you'd have to define a move constructor like:
```
FiniteElementVector::FiniteElementVector(FiniteElementVector&& input_vector)
    : mesh_(input_vector.mesh()),
      coll_(std::move(input_vector.coll_)),
      space_(std::move(input_vector.space_)),
      gf_(std::move(input_vector.gf_)),
      name_(std::move(input_vector.name_))
{
  auto *par_vec = input_vector.true_vec.StealParVector();
  true_vec.WrapHypreParVector(par_vec);
}
```
Or, define a proper move constructor for `mfem::HypreParVector`

Reply:
Yes, you are right! Thanks! (and sorry for the spam)

Reply:
No worries!